### PR TITLE
Add change log and release scripts

### PR DESCRIPTION
**Pull request by MartinWheelerMT**
## What

This adds a change log to the repository for the Github releases.
It also adds a release script, taking the steps required from the developer documentation, and updates the developer documentation to reflect this.

## Why

To ensure details of a releases are captured, which alongside the documentation and release script will help to improve the efficiency of the release process.

## Type of change

- [x] Internal change (non-breaking change with no effect on the functionality affecting end users)

## Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by ole4ryb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## [0.13] - 2023-09-13
+
+### Added
+
+Add details about object storage to operating documentation.
+
+### Fixed
+
+Fix issue where attachments with filenames containing special characters causes the transfer to fail.
```
Comment (adrianclay) [inline CHANGELOG.md:18]: I think these need bullet points, otherwise markdown will render them all as one paragraph.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
Comment (adrianclay) [inline CHANGELOG.md:5]: I'm not sure this is true given that `0.13` is not a valid Semantic version 😆 .

Though I'd be in favour of moving to semantic versions.

Comment (MartinWheelerMT) [reply CHANGELOG.md:5]: Removed the line and we will fix the versioning on the next release

File: CHANGELOG.md (hunk 1 above)
Comment (MartinWheelerMT) [reply CHANGELOG.md]: Have added bullet points

**Review by adrianclay — APPROVED** [review]

**Pull request merged**